pytorch_to_returnn/converter/converter.py

```diff
@@ -294,6 +294,7 @@ def _run_returnn_standalone_net_dict(self):
       config = Config({
         "extern_data": {"data": self._returnn_in_data_dict},
         "debug_print_layer_output_template": True,
+        "behavior_version": 12,
```

Comment:
I would not hardcode this number, but rather put it somewhere as a global, e.g. to `naming/returnn_ctx.py`.